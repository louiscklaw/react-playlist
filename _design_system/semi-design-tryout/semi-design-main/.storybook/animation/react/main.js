const config = require('../../base/base');

module.exports = {
  ...config,
  "stories": [
    '../../../packages/(semi-animation-react|semi-animation-styled)/_story/*.react.stories.(js|jsx)',
  ],
  typescript: {
    check: false,
    checkOptions: {}
  },
};
