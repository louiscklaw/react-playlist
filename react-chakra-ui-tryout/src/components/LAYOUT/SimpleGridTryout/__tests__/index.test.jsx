import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';

import { SimpleGridTryout } from '../';

describe('test SimpleGridTryout tryout', () => {
  test('helloworld', () => {
    const tree = renderer.create(<SimpleGridTryout />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
