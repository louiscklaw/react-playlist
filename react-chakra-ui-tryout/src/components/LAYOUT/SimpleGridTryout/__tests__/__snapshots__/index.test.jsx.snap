// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test SimpleGridTryout tryout helloworld 1`] = `
<div>
  SimpleGridTryout
  <div>
    <div
      className="css-1ky8x8j"
    >
      <div
        className="css-9rloc5"
      />
      <div
        className="css-9rloc5"
      />
      <div
        className="css-9rloc5"
      />
      <div
        className="css-9rloc5"
      />
      <div
        className="css-9rloc5"
      />
    </div>
  </div>
  <div>
    <div
      className="css-1238olr"
    >
      <div
        className="css-9rloc5"
      />
      <div
        className="css-9rloc5"
      />
      <div
        className="css-9rloc5"
      />
      <div
        className="css-9rloc5"
      />
      <div
        className="css-9rloc5"
      />
    </div>
  </div>
  <div>
    <div
      className="css-wb6fz"
    >
      <div
        className="css-9rloc5"
      />
      <div
        className="css-9rloc5"
      />
      <div
        className="css-9rloc5"
      />
      <div
        className="css-9rloc5"
      />
      <div
        className="css-9rloc5"
      />
      <div
        className="css-9rloc5"
      />
    </div>
  </div>
  <div>
    <div
      className="css-luuvyl"
    >
      <div
        className="css-9rloc5"
      />
      <div
        className="css-9rloc5"
      />
      <div
        className="css-9rloc5"
      />
      <div
        className="css-9rloc5"
      />
      <div
        className="css-9rloc5"
      />
    </div>
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
</div>
`;
